### ISSUE TITLE
Add support for global folder prefixes

### ISSUE THREAD

**Issue by rgrove** [rgrove/larch#39]
See https://groups.google.com/forum/#!topic/larch/jc7cllhbnD4

**Comment by jxchong** [comment]
Did this ever get implemented?

**Comment by rgrove** [comment]
Nope. I no longer actively develop or use Larch, and nobody has submitted a pull request that adds this feature.

**Comment by rahul286** [comment]
@rgrove 

> use Larch

Are you using something else or don't need to move mails as part of daily job routine? We use larch quite frequently so wondering.

**Comment by rgrove** [comment]
I just haven't had a need to move large amounts of email in many years.

**Comment by rahul286** [comment]
Great. I used this many times and can't thank you enough :-)

**Comment by rgrove** [comment]
That means a lot, @rahul286! I'm glad you found Larch useful.

**Comment by jxchong** [comment]
I found Larch because apparently Microsoft's support for "migrating" from our internal IMAP to Exchange Online/Offic365 consists of setting up email forwarding but not actual migrating the existing email content from one server to the other. While support for a global folder prefix would've helped a bit, I was able to mess around with folder structures enough to at least get my email migrated to office365... using Larch and only 11 emails failed to transfer (out of ~12k) because of the "Net::IMAP::ResponseParseError: unexpected token EOF (expected CRLF)"

So thanks so much!  Hopefully Larch will keep working for a while and/or someone will pick up development in the future. This was a way better option than relying on imapsync.
